cross-project-tests/debuginfo-tests/dexter/dex/command/commands/DexContinue.py

```diff
@@ -0,0 +1,44 @@
+# DExTer : Debugging Experience Tester
+# ~~~~~~   ~         ~~         ~   ~~
+#
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""A Command that tells dexter to set a breakpoint which, after hitting,
+signals that the debugger shuold 'continue' until another is hit. Continuing
+out of a function being stepped through with DexStepFunction is well defined:
+stepping will resume in other functions tracked further down the stacktrace.
+
+NOTE: Only supported for DAP based debuggers.
+"""
+
+from dex.command.CommandBase import CommandBase
+
+
+class DexContinue(CommandBase):
+    def __init__(self, *args, **kwargs):
+        if len(args) == 0:
+            self.expression = None
+            self.values = []
+        elif len(args) == 1:
+            raise TypeError("expected 0 or at least 2 positional arguments")
+        else:
+            self.expression = args[0]
+            self.values = [str(arg) for arg in args[1:]]
+        self.from_line = kwargs.pop("from_line", 1)
+        self.to_line = kwargs.pop("to_line", 999999)
```

Comment:
How do the arguments to this work? I would assume that this function would be willing to accept a `on_line` argument, and I'd also assume that in the absence of any line argument ("on" or "from/to") this command is essentially meaningless, so it should be necessary to pass at least one.

Reply:
Do you feel strongly that `on_line` is better than `from_line` when `to_line` is omitted? My preference is to keep it to just `from_line` as it still makes sense to me and keeps things simpler, but I don't mind changing it if you'd prefer.

I agree that we need at least one line arg - done.